client/my-sites/domains/domain-management/settings/cards/domain-status/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import {
+	createSiteFromDomainOnly,
+	domainManagementTransferToOtherSite,
+} from 'calypso/my-sites/domains/paths';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import type { DomainStatusCardProps } from './types';
+
+import './style.scss';
+
+const DomainStatusCard = ( props: DomainStatusCardProps ): JSX.Element => {
+	const translate = useTranslate();
+	const { selectedSite, selectedDomainName, currentRoute } = props;
+
+	return (
+		<div className="domain-status__card">
+			<div className="domain-status__card-content">
+				<p>{ translate( 'Your domain is registered but not pointing to a site.' ) }</p>
```

Comment:
I think this should say "This domain is not associated with a WordPress.com site." since the domain could be pointing to some non-WPCOM site.